python/lsst/ts/salobj/topics/read_topic.py

```diff
@@ -240,7 +246,7 @@ def __init__(
         name,
         sal_prefix,
         max_history,
-        queue_len=DDS_READ_QUEUE_LEN,
+        queue_len=DEFAULT_QUEUE_LEN,
```

Comment:
I wonder if we should really have a default value for this parameter. I think it is quite a critical and important parameter, and I think you are always setting it when you start no?

Reply:
I think most code uses the default.